microservices/s3_to_redshift/ddl/cmslite.group.sql

```diff
@@ -1,9 +1,9 @@
-CREATE TABLE IF NOT EXISTS cmslite.group
+CREATE TABLE IF NOT EXISTS test.cms_group
```

Comment:
This is a short lived test schema. I'm not sure why this should be merged to master.